allure-report-face/src/main/webapp/templates/testcase/attachment.html

```diff
@@ -19,7 +19,7 @@
             <pre><code>{{attachText}}</code></pre>
         </div>
         <div ng-switch-when="csv">
-            <table class="test-list table table-hover table-condensed">
+            <table class="test-list table table-hover table-striped table-condensed">
```

Comment:
Could you please remove the `table-hover` class? It causes hover effect on table rows, but we are using it only for clickable items.